docs/monitoring/monitoring-kibana.asciidoc

```diff
@@ -1,5 +1,100 @@
+[role="xpack"]
 [[monitoring-kibana]]
-== Monitoring Kibana
+== Configuring monitoring in {kib}
+++++
+<titleabbrev>Configuring monitoring</titleabbrev>
+++++
 
-See
-{kibana-ref}/monitoring-xpack-kibana.html[Monitoring {kib}]. 
+If you enable the collection of monitoring data in your cluster, you can 
+optionally collect metrics about {kib}. 
+
+The following method involves sending the metrics to {es}, which ultimately 
+routes them to the monitoring cluster. For an alternative method, see 
```

Comment:
> The following method involves sending the metrics to Elasticsearch, which ultimately routes them to the monitoring cluster.

"Elasticsearch" and "monitoring cluster" are both Elasticsearch clusters. So I wonder if this wording might be confusing.

In our internal discussions we often use the terms "production cluster" and "monitoring cluster" to draw the distinction between the two Elasticsearch clusters in the context of Monitoring. And, of course, users might use the same cluster for both purposes, meaning that their production and monitoring clusters are one and the same.

I'm not sure if this needs rewording but I thought I'd bring this up. Ultimately I trust your judgement on what best to do here 😄.